### ISSUE TITLE
Installation on addons.mozilla.org doesn't work on FreeBSD 

### ISSUE THREAD

**Issue by lantw44** [gnome-integration-team/firefox-gnome#336]
[The web page on addons.mozilla.org](https://addons.mozilla.org/zh-TW/firefox/addon/adwaita/) shows 'Not available for your platform' on FreeBSD. Although we can still manually install the extension by downloading the xpi file from GitHub, it seems automatic updates in Firefox is broken.

**Comment by seleznev** [comment]
It's very strange issue because we have following string in install.rdf (added in #265):

```
<em:targetPlatform>FreeBSD</em:targetPlatform>
```

Can you provide output of `uname -a` command from your's system?

**Comment by lantw44** [comment]
```
FreeBSD fbwn 10.1-RELEASE-p10 FreeBSD 10.1-RELEASE-p10 #0: Wed May 13 06:54:13 UTC 2015     root@amd64-builder.daemonology.net:/usr/obj/usr/src/sys/GENERIC  amd64
```

**Comment by seleznev** [comment]
@lantw44, well... What is yours user-agent?

**Comment by lantw44** [comment]
`Mozilla/5.0 (X11; FreeBSD amd64; rv:38.0) Gecko/20100101 Firefox/38.0`

**Comment by seleznev** [comment]
Well, I can reproduce issue with you User-agent. It looks like it's problem on AMO website side. But if I click on "disabled" install button I see link "Install for Linux anyway". May be it can help you.

Also, please try to set `general.useragent.override` to something with Linux (like `Mozilla/5.0 (X11; Linux x86_64; rv:41.0) Gecko/20100101 Firefox/41.0` for ex.) at `about:config` page.

**Comment by lantw44** [comment]
Yes, I can use "install for Linux anyway", but I guess I have to set `general.useragent.override` to make automatic updates work.

So we should move this bug to AMO ...

**Comment by lantw44** [comment]
Moved to here: https://bugzilla.mozilla.org/show_bug.cgi?id=1181958
